agir/front/components/toolsPage/index.js

```diff
@@ -0,0 +1,470 @@
+import React, { useState, useEffect } from "react";
+import axios from "@agir/lib/utils/axios";
+import style from "@agir/front/genericComponents/_variables.scss";
+import styled from "styled-components";
+
+import { Hide } from "@agir/front/genericComponents/grid";
+import Button from "@agir/front/genericComponents/Button";
+import Spacer from "@agir/front/genericComponents/Spacer";
+import Link from "@agir/front/app/Link";
+import { RawFeatherIcon } from "@agir/front/genericComponents/FeatherIcon";
+
+import img_JLM_2022 from "@agir/front/genericComponents/logos/JLM_2022.jpg";
+import img_AvenirEnCommun from "@agir/front/genericComponents/logos/AvenirEnCommun.jpg";
+import img_Linsoumission from "@agir/front/genericComponents/logos/Linsoumission.jpg";
+import img_Comparateur from "@agir/front/genericComponents/logos/Comparateur.jpg";
+import bricksNSP from "@agir/front/genericComponents/images/login_bg_desktop.svg";
+
+import Navigation from "@agir/front/dashboardComponents/Navigation";
+import nonReactRoutes from "@agir/front/globalContext/nonReactRoutes.config";
+import { routeConfig } from "@agir/front/app/routes.config";
+
+const Container = styled.div`
+  padding: 25px 85px;
+
+  @media (max-width: ${style.collapse}px) {
+    padding: 20px;
+  }
+`;
+
+const BlockTitle = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 23px;
+  margin-top: 23px;
+
+  > div:first-child {
+    display: flex;
+    align-items: center;
+  }
+
+  ${RawFeatherIcon} {
+    margin-right: 6px;
+  }
+`;
+
+const BlockContent = styled.div`
+  margin-bottom: 20px;
+`;
+
+const Title = styled.div`
+  font-weight: 700;
+  font-size: 26px;
+
+  @media (max-width: ${style.collapse}px) {
+    font-size: 20px;
+    font-weight: 600;
+  }
+`;
+
+const Subtitle = styled.div`
+  font-weight: 600;
+  font-size: 1rem;
+  margin: 20px 0;
+`;
+
+const StyledButton = styled(Button)`
+  height: 40px;
+  font-size: 1rem;
+  font-weight: 500;
+  border-radius: 0.5rem;
```

Comment:
On a une variable pour la nouvelle valeur du border-radius
```suggestion
  border-radius: ${(props) => props.theme.borderRadius};
```